### PR TITLE
chore: add meetings minutes for office hour on 17-11-2023

### PR DESCRIPTION
**Pull request by FaGru3n**
#### What would be changed or will be added with this PR?
- add office hour 17-11-2023

fixes https://github.com/eclipse-tractusx/sig-infra/issues/374

#### Preview
image

#### Testing 

- checkout branch
- npm install (optional)
- npm start
- verify changes

### CODE REVIEW

File: blog/2023-11-17-OfficeHours.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+## General updates / information
+
+- Idea that System Team Office Hour will become more open - Please follow upcoming mailing list infos and [Eclipse Matrix Chat](https://chat.eclipse.org/#/room/#tractusx:matrix.eclipse.org) discussions
+
+### DevSecOps / System-Team
+
+- _Info_: Quality Gate  (QG) issues from System Team will come soon
+- _Info_: Kyverno Policies will be getting in place planning to our clusters, and we're playing around with that
```
Comment (SebastianBezold) [inline blog/2023-11-17-OfficeHours.md:15]: "in place planning to our clusters"?? 
suggestion:

`We are going to deploy [Kyverno](https://kyverno.io/) to our clusters. Goal: Verify Release Guidelines at deploy time; Warnings at first; might be enforced in future`

File: blog/2023-11-17-OfficeHours.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- PostgresSQL Alignment required feedback - Please join the [discussion](https://github.com/eclipse-tractusx/sig-infra/discussions/308)
+- Package manager upgrades in Dockerfiles are a "no-go" see [TRG 4.2](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02)
+- Trivy workflow failing on HIGH findings see [here](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fmessage%2F19%3Ameeting_NTQxZjkzNTQtYTc4MC00NzQ3LWE2N2YtMzQ5YzMyM2E1MzMw%40thread.v2%2F1700223757067%3Fcontext%3D%257B%2522contextType%2522%253A%2522chat%2522%257D&type=message&deeplinkId=216bcb26-39f8-480d-9952-bee5e7cdf145&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
+- HashiCorp Vault tokens for EDC deployments --> Be careful with your root vault token in your application! Please use the Argo CD Vault plugin mechanism.
```
Comment (SebastianBezold) [inline blog/2023-11-17-OfficeHours.md:20]: Teams are not using the root token. This was just an example of the only token that does not expire. So just background info.

File: blog/2023-11-17-OfficeHours.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+### Security
+
+- Alternative to Invicti dynamic scanning with [OWASP ZAP](https://www.zaproxy.org/) in elaboration
+- GitGuardian secret scanning was done and security team get aware of HashiCorp Vault tokens problematic
+  - will be a meeting in place on January 2024 and technique will be presented
+- QG Process is ongoing
+- Token discussions will be supported by Security Team contact them under [sig-security repo](https://github.com/eclipse-tractusx/sig-security/issues/new/choose)
```
Comment (SebastianBezold) [inline blog/2023-11-17-OfficeHours.md:28]: by Security Team. Contact -> new sentence starting here

File: blog/2023-11-17-OfficeHours.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### FOSS
+
+- Congrats to [Andrea Bertagnolli](https://github.com/ndr-brt) was successfully voted to be a committer in the [Eclipse Tractus-X project](https://projects.eclipse.org/projects/automotive.tractusx)
+- YouTube Videos from [Eclipse Foundation Con](https://www.youtube.com/@EclipseFdn) are available
```
Comment (SebastianBezold) [inline blog/2023-11-17-OfficeHours.md:33]: It's EclipseCon not Eclipse Foundation Con

File: blog/2023-11-17-OfficeHours.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- Alternative to Invicti dynamic scanning with [OWASP ZAP](https://www.zaproxy.org/) in elaboration
+- GitGuardian secret scanning was done and security team get aware of HashiCorp Vault tokens problematic
+  - will be a meeting in place on January 2024 and technique will be presented
+- QG Process is ongoing
+- Token discussions will be supported by Security Team contact them under [sig-security repo](https://github.com/eclipse-tractusx/sig-security/issues/new/choose)
+
+### FOSS
+
+- Congrats to [Andrea Bertagnolli](https://github.com/ndr-brt) was successfully voted to be a committer in the [Eclipse Tractus-X project](https://projects.eclipse.org/projects/automotive.tractusx)
```
Comment (SebastianBezold) [inline blog/2023-11-17-OfficeHours.md:32]: on a successful election. or end sentence after name and start the new one with "He was successfully elected"

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: blog/2023-11-17-OfficeHours.md (hunk 1 above)
Comment (FaGru3n) [reply blog/2023-11-17-OfficeHours.md:15]: always good to have a second pair of eyes

File: blog/2023-11-17-OfficeHours.md (hunk 5 above)
Comment (FaGru3n) [reply blog/2023-11-17-OfficeHours.md]: [d62868a](https://github.com/catenax-ng/catenax-ng.github.io/pull/219/commits/d62868af87380365e2008292d9159bc3fa2a15b7)

File: blog/2023-11-17-OfficeHours.md (hunk 4 above)
Comment (FaGru3n) [reply blog/2023-11-17-OfficeHours.md]: [d62868a](https://github.com/catenax-ng/catenax-ng.github.io/pull/219/commits/d62868af87380365e2008292d9159bc3fa2a15b7)

File: blog/2023-11-17-OfficeHours.md (hunk 3 above)
Comment (FaGru3n) [reply blog/2023-11-17-OfficeHours.md]: [d62868a](https://github.com/catenax-ng/catenax-ng.github.io/pull/219/commits/d62868af87380365e2008292d9159bc3fa2a15b7)

Comment (FaGru3n) [reply blog/2023-11-17-OfficeHours.md]: https://github.com/catenax-ng/catenax-ng.github.io/pull/219/commits/95c24e5f1bff1c749ff9e740365ba50c419c51e3

**Comment by FaGru3n** [comment]
Thx to you @SebastianBezold

**Review by tomaszbarwicki — APPROVED** [review]
LGTM :)

**Pull request merged**